### PR TITLE
Add a service owner boolean to a user

### PR DESCRIPTION
**Pull request by pezholio**
This adds a new `serviceOwner` boolean to a user, allowing us to specify if a user can see content for all organisations, or just their organisation.

## Screnshots of UI changes

image

### CODE REVIEW

File: views/users/roles/edit.njk [hunk 1]
```diff
@@ -24,6 +25,31 @@
...
+            isPageHeading: false,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        hint: {
+          text: ("users.form.hint.serviceOwner" | t)
+        },
+        items: [
+          {
+            value: 1,
```
Comment (Gweaton) [inline views/users/roles/edit.njk:43]: I've got something similar in my branch (added a `yesAndNoRadioButtonsPresenter`) that uses "yes" and "no" for its values - what's the motivation behind 1 and 0 here? I can see the logic, I just don't think I've done it this way before.

File: src/professions/admin/professions.controller.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class ProfessionsController {
 
     const user = request['appSession'].user as User;
 
-    const overview = user.roles.includes(UserRole.Admin);
+    const overview = user.serviceOwner;
```
Comment (Gweaton) [inline src/professions/admin/professions.controller.ts:52]: Do you think we could we make this variable name a bit more boolean-sounding like `isOverview` or `showAllOrganisations`?

**Review by Gweaton — APPROVED** [review]
Looks good, just a couple of comments.

File: views/users/roles/edit.njk (hunk 1 above)
Comment (pezholio) [reply views/users/roles/edit.njk:43]: I guess it just seemed like a more logical way (to me anyway) to cast to a boolean. Again, proabably a carry-over from Rails.

File: src/professions/admin/professions.controller.ts (hunk 2 above)
Comment (pezholio) [reply src/professions/admin/professions.controller.ts:52]: Yeah, nice idea 👍

**Pull request merged**